benchmarks/community-benchmark/run.sh

```diff
@@ -50,6 +70,8 @@ else
 	export USE_CASE=1
 	mandatory BRANCH
```

Comment:
Can we rename `BRANCH` to `BASE`? That would feel more consistent, and would simplify the code below
```suggestion
	mandatory BASE
```

Reply:
Done